lucene-modules-integrationtests/wildflyConfig/standalone/configuration/standalone-lean.xml

```diff
@@ -0,0 +1,170 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
Is it really needed to maintain a server config? Alternatively, we can use whatever is in the server